README.md

```diff
@@ -151,35 +151,74 @@ to learn how to set up topology using this role.
 
 ## Role variables
 
+Configuration format is described in detail in the
+[configuration format](#configuration-format) section.
+
 Role variables are used to configure started instances, cluster topology,
 vshard bootstrapping, and failover.
 
-Configuration format is described in detail in the
-[configuration format](#configuration-format) section.
+Common variables:
 
-* `cartridge_package_path` (`string`, optional): path to Cartridge RPM package
-  (application name will be detected as package name);
 * `cartridge_app_name` (`string`): application name, required;
 * `cartridge_cluster_cookie` (`string`, required): cluster cookie for all
   cluster instances;
-* `cartridge_defaults` (`dict`, optional, default: `{}`): default configuration
-  parameters values for instances;
-* `cartridge_bootstrap_vshard` (`boolean`, optional, default: `false`): boolean
-  flag that indicates if vshard should be bootstrapped;
-* `cartridge_wait_buckets_discovery` (`boolean`, optional, default: `true`): boolean
-  flag that indicates if routers should wait for buckets discovery after vshard bootstrap;
-* `cartridge_failover_params` (`dict`, optional): [failover](#failover) parameters;
-* `cartridge_app_config` (`dict`, optional): application config sections to patch;
-* `cartridge_auth`: (`dict`, optional): [authorization configuration](#cartridge-authorization);
-* `cartridge_enable_tarantool_repo` (`boolean`, optional, default: `true`):
-  indicates if the Tarantool repository should be enabled (for packages with
-  open-source Tarantool dependency);
+
+Role scenario configuration:
+
 * `cartridge_scenario` (`list-of-strings`, optional): list of steps to be launched
   (see [change scenario](#using-scenario) for more details)
 * `cartridge_custom_steps_dir` (`string`, optional, default: `null`): path to directory
   containing YAML files of custom steps (see [change scenario](#using-scenario) for more details)
 * `cartridge_custom_steps` (`list-of-dicts`, optional, default: `[]`): list of custom steps
   (see [change scenario](#using-scenario) for more details)
+
+Application package configuration:
+
+* `cartridge_package_path` (`string`, optional): path to Cartridge RPM package
+  (application name will be detected as package name);
+* `cartridge_enable_tarantool_repo` (`boolean`, optional, default: `true`):
+  indicates if the Tarantool repository should be enabled (for packages with
+  open-source Tarantool dependency);
+
+[TGZ](./doc/tgz.md)-specific configuration:
```

Comment:
```suggestion
[TGZ](/doc/tgz.md)-specific configuration:
```

Reply:
Why?

Reply:
It's a full path, it's works in any place in project